lib/rules/no-restricted-imports.js

```diff
@@ -64,22 +64,26 @@ module.exports = {
 
             everything: "* import is invalid because '{{importNames}}' from '{{importSource}}' is restricted.",
             // eslint-disable-next-line eslint-plugin/report-message-format
-            everythingWithCustomMessage: "* import is invalid because '{{importNames}}' from '{{importSource}}' is restricted. {{customMessage}}"
+            everythingWithCustomMessage: "* import is invalid because '{{importNames}}' from '{{importSource}}' is restricted. {{customMessage}}",
+
+            importName: "'{{importName}}' import from '{{importSource}}' is restricted.",
+            // eslint-disable-next-line eslint-plugin/report-message-format
+            importNameWithCustomMessage: "'{{importName}}' import from '{{importSource}}' is restricted. {{customMessage}}"
         },
 
         schema: {
             anyOf: [
                 arrayOfStringsOrObjects,
                 {
                     type: "array",
-                    items: [{
+                    items: {
                         type: "object",
                         properties: {
                             paths: arrayOfStringsOrObjects,
                             patterns: arrayOfStrings
                         },
                         additionalProperties: false
-                    }],
+                    },
```

Comment:
These `[]` brackets were recently added in #12639 to avoid a possible misconfiguration. Did this change cause some other problems? Asking because you're removing them now, so I'm guessing there were some issues with the tests from this PR.

Reply:
Ooops... I think I lost it while rebasing...